src/components/RocketCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import styled from 'styled-components';
+import ReserveButton from './ReserveButton';
+import LeaveButton from './LeaveButton';
+import ReserveBadge from './ReserveBadge';
+
+const Wrapper = styled.div`
+  display: flex;
+  column-gap: 1em;
+  padding: 1em 2em;
+
+  img {
+    width: 100%;
+  }
+
+  h3 {
+    margin-top: 0;
+  }
+
+  p {
+    margin-top: 0;
+  }
+`;
+
+const ImgDiv = styled.div`
+  flex-grow: 1;
+  max-width: 15em;
+`;
+
+const ContentDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  flex-grow: 3;
+`;
+
+const RocketCard = (props) => {
+  const {
+    id, rocketName, description, flickrImages, reserved,
+  } = props;
+  return (
+    <Wrapper>
+      <ImgDiv>
+        <img src={flickrImages} alt="" />
+      </ImgDiv>
+      <ContentDiv>
+        <h3>{rocketName}</h3>
+        <p>
+          <span>
+            {(() => {
+              if (reserved) {
+                return <ReserveBadge />;
+              }
+              return '';
+            })()}
+          </span>
```

Comment:
- [ ] Nice idea with the IIME implementation here. However, as stated in the requirements you are required to use a specific React conditional syntax for your rendering.

  ![anwar-condit](https://user-images.githubusercontent.com/36542986/162099237-8172f1fa-49c8-4750-abfa-fc59c355c161.jpg)

  **SUGGESTION:** You could refactor to something like so:
  ```
  <span>
    {reserved && (
      <ReserveBadge />
    )}
    {!reserved && (
      ''
    )}
  </span>
  ```